packages/TesterApp/android/gradle.properties

```diff
@@ -41,4 +41,7 @@ newArchEnabled=false
 
 # Use this property to enable or disable the Hermes JS engine.
 # If set to false, you will be using JSC instead.
-hermesEnabled=true
+# NOTE: This is **NOT** set to opt-out of the automatic Hermes transformation
+# performed by the react gradle plugin. Instead, this transformation happens
+# during the webpack build.
+# hermesEnabled=false
```

Comment:
This happens to skip the main chunk hermes compilation performed in [BundleHermesCTask.kt](https://github.com/facebook/react-native/blob/main/packages/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/tasks/BundleHermesCTask.kt#L92-L113). Setting this to `true` would enable it, and setting it to `false` would end up excluding the hermes runtime from the build.

Another option is setting this to `false` (matching the iOS change) and adding setting [enableSoCleanup](https://github.com/facebook/react-native/blob/main/packages/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/ReactExtension.kt#L81) to `false`. This should result in a skipped transformation but allows us to still include the hermes runtime.